collector/aspnet_aspnetapplications.go

```diff
@@ -0,0 +1,1239 @@
+	ErrorsTotal                           uint32
+	ErrorsTotalPerSec                     uint32
+	ErrorsUnhandledDuringExecution        uint32
+	ErrorsUnhandledDuringExecutionPerSec  uint32
+	EventsRaised                          uint32
+	EventsRaisedPerSec                    uint32
+	FormsAuthenticationFailure            uint32
+	FormsAuthenticationSuccess            uint32
+	InfrastructureErrorEventsRaised       uint32
+	InfrastructureErrorEventsRaisedPerSec uint32
+	ManagedMemoryUsedestimated            uint32
+	MembershipAuthenticationFailure       uint32
+	MembershipAuthenticationSuccess       uint32
+	OutputCacheEntries                    uint32
+	OutputCacheHitRatio                   uint32
+	OutputCacheHits                       uint32
+	OutputCacheMisses                     uint32
+	OutputCacheTrims                      uint32
+	OutputCacheTurnoverRate               uint32
+	PercentManagedProcessorTimeestimated  uint32
+	PipelineInstanceCount                 uint32
+	RequestBytesInTotal                   uint32
+	RequestBytesInTotalWebSockets         uint32
+	RequestBytesOutTotal                  uint32
+	RequestBytesOutTotalWebSockets        uint32
+	RequestErrorEventsRaised              uint32
+	RequestErrorEventsRaisedPerSec        uint32
+	RequestEventsRaised                   uint32
+	RequestEventsRaisedPerSec             uint32
+	RequestExecutionTime                  uint32
+	RequestsDisconnected                  uint32
+	RequestsExecuting                     uint32
+	RequestsExecutingWebSockets           uint32
+	RequestsFailed                        uint32
+	RequestsFailedWebSockets              uint32
+	RequestsInApplicationQueue            uint32
+	RequestsNotAuthorized                 uint32
+	RequestsNotFound                      uint32
+	RequestsPerSec                        uint32
+	RequestsRejected                      uint32
+	RequestsSucceeded                     uint32
+	RequestsSucceededWebSockets           uint32
+	RequestsTimedOut                      uint32
+	RequestsTotal                         uint32
+	RequestsTotalWebSockets               uint32
+	RequestWaitTime                       uint32
+	SessionsAbandoned                     uint32
+	SessionsActive                        uint32
+	SessionSQLServerconnectionstotal      uint32
+	SessionStateServerconnectionstotal    uint32
+	SessionsTimedOut                      uint32
+	SessionsTotal                         uint32
+	TransactionsAborted                   uint32
+	TransactionsCommitted                 uint32
+	TransactionsPending                   uint32
+	TransactionsPerSec                    uint32
+	TransactionsTotal                     uint32
+	ViewstateMACValidationFailure         uint32
+}
+
+func (c *aspnet_aspnetapplicationsCollector) collect(ch chan<- prometheus.Metric) (*prometheus.Desc, error) {
+	var dst []Win32_PerfRawData_ASPNET_ASPNETApplications
+	q := queryAll(&dst)
+	if err := wmi.Query(q, &dst); err != nil {
+		return nil, err
+	}
+
```

Comment:
I would expect there is data returned for each application? So we'd need to loop here and add the name as a label.